website/content/docs/upgrade/v1_11.mdx

```diff
@@ -3,82 +3,73 @@ page_title: Upgrade to v1.11
+  Although you can add non-GitHub sources to `required_plugins` for validation purposes, remote installation is still limited to Github-sourced plugins. Refer to [Define plugin source](/packer/docs/templates/hcl_templates/blocks/packer#define-plugin-source) in the `required_plugins` reference documentation for additional information.
+
+## Troubleshooting
 
 This section provides guidance on potential issues you may experience after upgrading.
 
-## Missing plugins error
+### Missing plugins error
 
 Packer may report that you one or more plugins are missing after the upgrade.
-This might happen if you had installed the plugin using one of the now unsupported methods of installation:
+This may occur if you had used one of the following plugin installation methods that are longer supported:
 
-* in the directory Packer is invoked from
-* alongside the Packer binary
-* under the root of the `PACKER_PLUGIN_PATH` directory
+* Installed the plugin to the directory Packer is invoked from.
+* Installed the plugin alongside the Packer binary.
+* Installed the plugin under the root of the `PACKER_PLUGIN_PATH` directory.
 
 For any missing components, you may need to reinstall their respective plugins as described in [Installing Plugins](/packer/docs/plugins/install).
 
-## Plugin local installation
+### Use `required_plugins` message
 
-If you cannot remotely install a plugin for any reason (firewalls, Github blocked, etc.), you can use the local installation methods.
-After downloading the plugin binary, you can manually install the plugin binary using `packer plugins install --path` command.
-This will place the plugin binary in the location that Packer expects it to be, with the correct naming scheme and sumfile.
+Packer may print a message directing you to specify plugins in the `required_plugins` block in your template and install them using the `packer init` command.
+This is the preferred way to manage the plugins, but it is not required. You can still install plugins manually using the `packer plugins install` command. Refer to [Installing Plugins](/packer/docs/plugins/install)
 
-## Using `required_plugins`
+### Blocked remote access to plugin 
 
-In some cases, Packer will hint at using `required_plugins` block in your template.
-With such a block, this opens the possibility to use `packer init` to automatically install plugins, and check that your environment is ready to build your template.
-While this is the preferred way to manage the plugins you need in order to build your images, this is not a hard requirement.
-You can still install plugins independently of a template by using the `packer plugins install` command variants.
+If you cannot remotely install a plugin for any reason, such as a firewall configuration or GitHub is blocking access, you can use the local installation methods. Refer to [Installing Plugins](/packer/docs/plugins/install)
```

Comment:
Do we have an anchor to the local installation method? It looks like the sentence before provides direction on where to go.

Reply:
Subsections are automatically anchors, so I believe it is possible.
That being said it doesn't need to happen now, it can be updated in a subsequent PR if needed